docs/ford.md

```diff
@@ -0,0 +1,20 @@
+---
+project: x3d2
+src_dir: ../src
+summary: x3d2 is a suite of high-order finite difference flow solvers dedicated
+         to the study of turbulent flows. x3d2 is a re-implementation of the Xcompact3d
+         framework to run on supercomputers (both GPUs and CPUs).
+author: Xcompact3d team
+exclude: cuda_allocator.f90
+         vector3d_cuda.f90
+         vector3d_simd.f90
+         vector3d.f90
```

Comment:
These source files don't actually exist in the codebase anymore so these lines can be removed.